paradise.dme

```diff
@@ -14,6 +14,7 @@
 #include "_maps\__MAP_DEFINES.dm"
 #include "_maps\base_map.dm"
 #include "_maps\map_datums.dm"
+#include "_maps\map_files\cyberiad\cyberiad.dmm"
```

Comment:
I dont think .dmm files are supposed to be included in the dme

Reply:
how did I even